apps/docs/src/app/core/component-docs/fixed-card-layout/examples/fixed-card-layout-examples.component.html

```diff
@@ -0,0 +1,85 @@
+<fd-fixed-card-layout>
+    <fd-card *fdCardDef style="color: blueviolet;" [header]="'1'">
```

Comment:
unnecessary binding
please check the whole file

Reply:
fixed